src/Client/Web/Bit.Client.Web.BlazorUI/Components/Labels/BitLabel.razor.cs

```diff
@@ -6,8 +6,14 @@ public partial class BitLabel
     {
         private bool isRequired;
 
+        /// <summary>
+        /// The for attribute specifies which form element a label is bound to
+        /// </summary>
         [Parameter] public string? For { get; set; }
 
+        /// <summary>
+        /// Whether the label is required or not
```

Comment:
the label itself cant be required, this parameter applies a specific style to the component which should be described in this comment.